ios/RNSentry.mm

```diff
@@ -379,24 +379,20 @@ - (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAdd
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
 #if SENTRY_HAS_UIKIT
-    SentryAppStartMeasurement *appStartMeasurement = PrivateSentrySDKOnly.appStartMeasurement;
-
-    if (appStartMeasurement == nil) {
+    NSDictionary<NSString *, id> *measurements = [PrivateSentrySDKOnly appStartMeasurementWithSpans];
```

Comment:
appStartMeasurementWithSpans is missing?

Reply:
I saw that you added it on https://github.com/getsentry/sentry-cocoa/pull/4021 but a new release was released before the inclusion of it.
We should wait for a new release of sentry cocoa before merging this PR

Reply:
Yes. Exactly. 👍